_includes/azerothcore/sidebar.html

```diff
@@ -140,4 +141,4 @@
         
```

Comment:
add on line 108


```suggestion
       <li><a href="/wiki/es/guide-to-triaging">Guía para la evaluación de errores</a></li> 
```

 for the Spanish translation